packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -43,6 +43,11 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       sipUri, deviceUrl, locusUrl, resourceId, correlationId, ensureConversation, moderator, pin, moveToResource, roapMessage
     } = options;
 
+    LoggerProxy.logger.log(
+      `${MeetingRequest.name}#${this.joinMeeting.name} --> Joining a meeting`,
```

Comment:
I believe the format should be `folder:file#method --> description`:

```js
LoggerProxy.logger.log(`Meeting:request#joinMeeting --> joining ${MeetingRequest.name}#${this.joinMeeting.name}`
```

Let me know if this is not the case for the scope of this JIRA.

Reply:
@InteractiveTimmy is correct

Reply:
@mwegman unresolving this convo because the output would be `MeetingRequest#joinMeeting --> Joining a meeting` which does not fit the format of `folder:file#method --> description`

Reply:
Please just use the extra verbose method for now
`Meeting:request#joinMeeting --> Joining a meeting`